hathor/nanocontracts/context.py

```diff
@@ -149,13 +183,12 @@ def to_json(self) -> dict[str, Any]:
                 caller_id = get_address_b58_from_bytes(self.caller_id)
             case ContractId():
                 caller_id = self.caller_id.hex()
-            case _:
+            case _:  # pragma: no cover
                 assert_never(self.caller_id)
 
         return {
             'actions': [action.to_json() for action in self.__all_actions__],
             'caller_id': caller_id,
-            'timestamp': self.timestamp,
```

Comment:
Maybe keep this to ease the transition too. Unless we're sure we can remove it with no breakage. Maybe @pedroferreira1 would know better.